frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -84,6 +84,15 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
       },
       replicas: 1,
     },
+    serverless: {
+      enabled: false,
+      scaling: {
+        minpods: 0,
+        maxpods: '',
+        concurrencyrequest: 100,
+        concurrencylimit: '',
```

Comment:
Should we be setting defaults? Even if the user doesn't open the advanced options these values will be set.

Please confirm if this expected.

cc @serenamarie125

Reply:
@parvathyvr ^^

Reply:
@christianvogt that's right! According to my conversation with @markusthoemmes ,Only Min Pods will be defaulted to 0...Rest of the fields can be left blank.For the blank fields,even if the user enter any values,that'd override anything the operator would set when installing Knative.
@invincibleJai @serenamarie125 FYI

Reply:
done